src/sugar3/graphics/toolcombobox.py

```diff
@@ -18,19 +18,43 @@
 """
 STABLE.
 """
-
 from gi.repository import Gtk
 from gi.repository import GObject
 
 from sugar3.graphics.combobox import ComboBox
 from sugar3.graphics import style
+'''
+A ToolComboBox is a widget that allows the user to
+choose from a list of valid choices.
```

Comment:
Example would be great here.  Try searching around activities to find if this is used.  Then you can adapt the code you find to make a standalone example.